Update intro article (#7)
AdguardTeam/KnowledgeBase: docs/intro.md

slug: /
---

> ## What is AdGuard Ad Blocker

Reviewer: What AdGuard Ad Blocker is / What is AdGuard Ad Blocker?

> AdGuard is one of the leaders on the market of ad blocking software with 10+ years of experience, almost a dozen products for various platforms, and over 30 million user installs. Unlike other ad blocking solutions, most of which are browser-based, we offer standalone apps for all major platforms, including mobile apps. They are more powerful than browser extensions, can block more and do it more efficiently:

Reviewer: ```suggestion
AdGuard is one of the leaders on the market of ad-blocking software with 10+ years of experience, almost a dozen products for various platforms, and 30+ million users. Unlike other ad-blocking solutions, most of which are browser-based, we offer standalone apps for all major platforms, both desktop and mobile ones. Apps are more powerful than browser extensions, can block more and do it more efficiently:
```

Author: там именно установки, кажется

[AdGuard for Windows](placeholder-link)
[AdGuard for Mac](placeholder-link)
[AdGuard for Android](placeholder-link)
[AdGuard for iOS](placeholder-link)

But we also develop and maintain free browser extensions for all popular browsers:

> [AdGuard Browser extension for Chromium, Firefox, Edge](placeholder-link)

Reviewer: Chrome?

Author: Не только Chrome, но и другие браузеры на основе хрома типа Оперы

Reviewer: тогда я бы написала Chromium-based browsers
